generators/server/templates/src/main/java/package/aop/logging/_LoggingAspect.java

```diff
@@ -51,7 +51,9 @@ public LoggingAspect(Environment env) {
     /**
      * Pointcut that matches all repositories, services and Web REST endpoints.
      */
-    @Pointcut("within(<%=packageName%>.repository..*) || within(<%=packageName%>.service..*) || within(<%=packageName%>.web.rest..*)")
+    @Pointcut("(within(io.github.jhipster.travis.repository..*) && @annotation(org.springframework.stereotype.Repository))"+
```

Comment:
you should keep `<%=packageName%>` instead of using fixed `io.github.jhipster.travis`